### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: src/main/java/taxi/controller/car/CreateCar.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/car/create.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String model = req.getParameter("model");
```
Comment (Max-Pochepets) [inline src/main/java/taxi/controller/car/CreateCar.java]:
```suggestion
        String model = req.getParameter("car_model");
```
name all your parameters in this manner

File: src/main/java/taxi/controller/car/DeleteCar.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class DeleteCar extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService = (CarService)
+            injector.getInstance(CarService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws IOException {
+        carService.delete(Long.parseLong(req.getParameter("id")));
```
Comment (Max-Pochepets) [inline src/main/java/taxi/controller/car/DeleteCar.java]:
```suggestion
        carService.delete(Long.parseLong(req.getParameter("car_id")));
```

File: src/main/webapp/WEB-INF/views/manufacturer/create.jsp [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Create manufacturer</title>
+</head>
+<body>
+<h1>Please enter data about manufacturer</h1>
+<form method="post" action="${pageContext.request.contextPath}/manufacturer/create">
+    Please provide manufacturer name<input type="text" name="name">
```
Comment (Max-Pochepets) [inline src/main/webapp/WEB-INF/views/manufacturer/create.jsp]:
```suggestion
    Please provide manufacturer name<input type="text" name="name" required>
```

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/taxi/controller/car/AddDriverToCar.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.model.Driver;
+import taxi.service.CarService;
+import taxi.service.DriverService;
+
+public class AddDriverToCar extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/taxi/controller/car/AddDriverToCar.java]:
```suggestion
public class AddDriverToCarController extends HttpServlet {
```

File: src/main/java/taxi/controller/car/CreateCar.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+import taxi.service.ManufacturerService;
+
+public class CreateCar extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/taxi/controller/car/CreateCar.java]: everywhere

File: src/main/webapp/web.xml [hunk 6]
```diff
@@ -3,13 +3,94 @@
...
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+    
+    <servlet>
+        <servlet-name>createDriver</servlet-name>
+        <servlet-class>taxi.controller.driver.CreateDriver</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>createDriver</servlet-name>
+        <url-pattern>/driver/create</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]: https://mate-academy.github.io/jv-program-fulltime/02_jdbc/checklist/08_servlets_practice_checklist.html
check all of your url patterns

File: src/main/webapp/web.xml [hunk 7]
```diff
@@ -3,13 +3,94 @@
...
+    </servlet-mapping>
+    <servlet>
+        <servlet-name>getAllDrivers</servlet-name>
+        <servlet-class>taxi.controller.driver.GetAllDrivers</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getAllDrivers</servlet-name>
+        <url-pattern>/driver/all</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
        <url-pattern>/drivers</url-pattern>
```

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**